File: ts/delivery-api/delivery_api_get_items.ts
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { createDeliveryClient } from '@kontent-ai/delivery-sdk';
import { Article } from './models/Article';

const deliveryClient = createDeliveryClient({
  environmentId: 'KONTENT_AI_ENVIRONMENT_ID'
});

const response = await deliveryClient.items<Article>()
  .type('article')
  .limitParameter(3)
  .toPromise();

File: ruby/delivery-api/delivery_api_get_items.rb
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kontent::Ai::Delivery::DeliveryClient.new project_id: 'KONTENT_AI_ENVIRONMENT_ID'
delivery_client.items('system.type'.eq('article'))
               .limit(3)
               .execute do |response|
                 items = response.items
               end
